### PR TITLE
fix gettingstarted files

### PR DESCRIPTION
**Pull request by finp**
## Motivation
reduce getting started, queries and mutations doc to minimal content

### CODE REVIEW

**Comment by finp** [comment]
@StephenCoady I'm having problems getting the mutation to work, could I ask you to take a look?
(works in playground, but not in client app)

File: modules/ROOT/pages/_partials/data-sync/js-client-mutate.adoc [hunk 1]
```diff
@@ -37,29 +28,30 @@ async function addPerson() {
   let client = await offlineClient.init();
 
   // Execute the mutation
-  client.query({
+  client.mutate({
       fetchPolicy: 'network-only',
```
Comment (StephenCoady) [inline modules/ROOT/pages/_partials/data-sync/js-client-mutate.adoc]: @finp I think this link might be your problem - probably left over from the query that was here. remove this and it should work.

Comment (jstaffor) [reply modules/ROOT/pages/_partials/data-sync/js-client-mutate.adoc]: @finp - I tested this locally and it works, :+1: @StephenCoady 

I also modified it to write to the html page using:

 document.write(JSON.stringify(data))

Screenshot from 2019-07-08 15-57-55

Comment (finp) [reply modules/ROOT/pages/_partials/data-sync/js-client-mutate.adoc]: @StephenCoady  thanks that was the trick. BTW we don't have a cli way to guide users thru this.
@jstaffor  I'm going with console cos a) we don't have a cli way of getting output b) we ask users later to use console https://docs.aerogear.org/aerogear/latest/ds-conflict.html#listening-to-conflicts

Comment (jstaffor) [reply modules/ROOT/pages/_partials/data-sync/js-client-mutate.adoc]: :+1:

**Pull request merged**